src/SPC/builder/unix/library/pkgconfig.php

```diff
@@ -11,7 +11,7 @@ trait pkgconfig
     protected function build(): void
     {
         $cflags = PHP_OS_FAMILY !== 'Linux' ? "{$this->builder->arch_c_flags} -Wimplicit-function-declaration -Wno-int-conversion" : '';
-        $ldflags = !($this instanceof LinuxLibraryBase) || $this->builder->libc === 'glibc' ? '' : '--static';
+        $ldflags = !($this instanceof LinuxLibraryBase) || getenv('SPC_LIBC') === 'glibc' ? '' : '--static';
```

Comment:
same question as before, can't it always be --static?

Reply:
Same as icu